registry-resolver/src/lib.rs

```diff
@@ -15,6 +17,10 @@ impl RegistryResolver {
 }
 #[async_trait::async_trait]
 impl ssi::DIDResolver for RegistryResolver {
+    fn get_method() -> String {
```

Comment:
this can be [`const fn`](https://doc.rust-lang.org/std/keyword.const.html#compile-time-evaluable-functions)

Reply:
@mkatychev I don't think a function in a trait can be a const, at least I get an error when I try

Reply:
@LuisOsta yes you'll have to make it a standalone function and call it from within the method.

Reply:
I don't understand, what's the value of doing that?

Reply:
Interesting, if we get the trait method `get_method` and just call a const function, why is it cheaper? Like how does that work?

Reply:
I've implemented the helper function @mkatychev , though some more context as to how this pattern actually works would be great